Yank/experiment.py

```diff
@@ -353,6 +359,7 @@ class ExperimentBuilder(object):
         'platform': 'fastest',
         'precision': 'auto',
         'switch_experiment_interval': 0,
+        'nodes_per_experiment': None
```

Comment:
Should we document the YAML option `nodes_per_experiment` in the docs?

Does it represent *nodes* or *MPI processes = GPUs* used per experiment?